docs/how_to_guides/how_to_create_surrogate_models.rst

```diff
@@ -0,0 +1,179 @@
+            expr=self.annual_energy
+            == -1 * self.electricity
+            * pyunits.convert(1 * pyunits.year, to_units=pyunits.hour)
+        )
+
+Create Training Data
+
+.. testcode::
+
+  def get_training_validation(self):
+    self.dataset_filename = os.path.join(
+        os.path.dirname(__file__), "data/dataset.pkl"
+    )
+    print('Loading Training Data...\n')
+    time_start = time.process_time()
+    pkl_data = pd.read_pickle(self.dataset_filename)
+    data = pkl_data.sample(n=int(len(pkl_data)))
+    self.data_training, self.data_validation = split_training_validation(
+        data, self.training_fraction, seed=len(data)
+    )
+    time_stop = time.process_time()
+    print("Data Loading Time:", time_stop - time_start, "\n")
+
+Create Surrogate
+
+.. testcode::
+
+  def create_surrogate(self):
+    self.training_fraction = 0.8 # Fraction of the sampled data to split for training and validation
+
+    self.get_training_validation()
+    time_start = time.process_time()
+
+    # Create PySMO trainer object
+    trainer = PysmoRBFTrainer(
+        input_labels=self.input_labels,
+        output_labels=self.output_labels,
+        training_dataframe=self.data_training,
+    )
+
+    # Set PySMO options
+    trainer.config.basis_function = "gaussian"  # default = gaussian
+    trainer.config.solution_method = "algebraic"  # default = algebraic
+    trainer.config.regularization = True  # default = True
+
+    # Train surrogate
+    rbf_train = trainer.train_surrogate()
+
+    # Create callable surrogate object
+    xmin, xmax = [self.design_size.bounds[0]], [self.design_size.bounds[1]]
+    input_bounds = {
+        self.input_labels[i]: (xmin[i], xmax[i]) for i in range(len(self.input_labels))
+    }
+    rbf_surr = PysmoSurrogate(rbf_train, self.input_labels, self.output_labels, input_bounds)
+
+    # Save model to JSON
+    if self.surrogate_file is not None:
+        print(f'Writing surrogate model to {self.surrogate_file}')
+        model = rbf_surr.save_to_file(self.surrogate_file, overwrite=True)
+
+Load the Surrogate
+
+.. testcode:: 
+
+  def load_surrogate(self):
+    print('Loading surrogate file...')
+    self.surrogate_file = os.path.join(
+        os.path.dirname(__file__), "pv_surrogate.json"
+    )
+
+    if os.path.exists(self.surrogate_file):
+
+        self.surrogate_blk = SurrogateBlock(concrete=True)
+        self.surrogate = PysmoSurrogate.load_from_file(self.surrogate_file)
+        self.surrogate_blk.build_model(
+            self.surrogate,
+            input_vars=self.surrogate_inputs,
+            output_vars=self.surrogate_outputs,
+        )
+
+Use the surrogate
+
+.. testcode:: 
+
+  m = ConcreteModel()
+  m.fs = FlowsheetBlock(dynamic=False)
+  m.fs.pv = PVSurrogate()
+  m.fs.pv.create_surrogate(save=True)
+
+  m.fs.pv.load_surrogate()
+
+  results = m.fs.pv.surrogate.evaluate_surrogate(
+      m.fs.pv.data_validation[m.fs.pv.input_labels]
+  )
```

Comment:
`evaluate_surrogate` wasn't mentioned previously. I guess it may be self-explanatory but given that create_ and load_surrogate are provided above, I figured `evaluate_surrogate` should be shown too.

Reply:
Ok I added the source code for the evaluate_surrogate function and added some context explaining that it is a built-in function of the PysmoSurrogate base class